rdf-star-cg-spec.html

```diff
@@ -343,11 +345,79 @@ <h2>Parsing</h2>
 
       <p>All other productions MUST be handled as specified by <a data-cite="TURTLE#h2_sec-parsing">Section 7 of the Turtle specification</a> [[TURTLE]], while still applying the changes above recursively.</p>
     </section>
+  </section>
+
+    <section id="n-triples-star">
+    <h2>N-Triples*</h2>
+
+    <p>This section describes N-Triples*, a minimal extension of the N-Triples format [[N-TRIPLES]] allowing a <a>subject</a> or an <a>object</a> of an <a>RDF* triple</a> to be an <a>embedded triple</a>. Note that there is no annotation syntax in N-Triples*.</p>
```

Comment:
```suggestion
    <p>This section describes N-Triples*, a minimal extension of the N-Triples format [[N-TRIPLES]] allowing a <a>subject</a> or an <a>object</a> of an <a>RDF* triple</a> to be an <a>embedded triple</a>.</p>
```
This sentence is also present in the Note below, I think it is enough.